app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const { getTopics } = require("./controller/topics-controller")
+const app = express();
+
+app.use(express.json())
```

Comment:
This is not being used for this question and so should be removed until its needed